src/presence_analyzer/static/presence_start_end.html

```diff
@@ -96,33 +94,39 @@
                     if(selected_user) {
                         loading.show();
                         chart_div.hide();
-                        
+                        $.getJSON("/api/v1/presence_start_end/"+selected_user,
+                            function(result) {
+                                final_result = [
+                                ["Mon", new Date(Date.parse("1,1,1,"+result["Monday"]["start"])),
+                                        new Date(Date.parse("1,1,1,"+result["Monday"]["end"]))],
+                                ["Tue", new Date(Date.parse("1,1,1,"+result["Tuesday"]["start"])),
+                                        new Date(Date.parse("1,1,1,"+result["Tuesday"]["end"]))],
+                                ["Wed", new Date(Date.parse("1,1,1,"+result["Wednesday"]["start"])),
+                                        new Date(Date.parse("1,1,1,"+result["Wednesday"]["end"]))],
+                                ["Thu", new Date(Date.parse("1,1,1,"+result["Thursday"]["start"])),
+                                        new Date(Date.parse("1,1,1,"+result["Thursday"]["end"]))],
+                                ["Fri", new Date(Date.parse("1,1,1,"+result["Friday"]["start"])),
+                                        new Date(Date.parse("1,1,1,"+result["Friday"]["end"]))],
+                                ]
```

Comment:
Also it would be a good idea to move all calculations to some util function to not to mess in the view - it should stay as clean as possible.